actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseEnrollmentActor.java

```diff
@@ -107,6 +109,20 @@ private void enrollCourseBatch(Request actorMessage) {
           LoggerEnum.INFO.name());
       UserCoursesService.sync(courseMap, (String) courseMap.get(JsonKey.ID));
     }
+    if (PropertiesCache.getInstance().getProperty(JsonKey.COURSE_BATCH_NOTIFICATIONS_ACTIVE) != null
```

Comment:
move complete code base to separate method.

Reply:
done